integration/examples/ruby/backend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ruby:2.7
+
+WORKDIR /app
+COPY . .
```

Comment:
Can we do something like:
 + Copy Gemfile*
 + Download the dependencies
 + Copy the rest of the sources

Reply:
Hey, @dgageot , thanks for review!
It's done.